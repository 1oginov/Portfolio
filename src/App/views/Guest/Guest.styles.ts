import { withStyles } from '@material-ui/core';

export default withStyles(theme => ({
  root: {
    alignItems: 'center',
    display: 'flex',
    flexDirection: 'column',
    flexGrow: 1,
    justifyContent: 'center',
  },
  row: {
    margin: theme.spacing(),
  },
}));
