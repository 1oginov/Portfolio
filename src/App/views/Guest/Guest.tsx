import * as React from 'react';

import GoogleLoginButton from 'User/components/GoogleLoginButton';

import Logo from '../../components/Logo';

interface Props {
  classes: { [key: string]: string };
}

const Guest: React.FunctionComponent<Props> = ({ classes }: Props) => (
  <div className={classes.root}>

    <div className={classes.row}>
      <Logo />
    </div>

    <div className={classes.row}>
      <GoogleLoginButton />
    </div>

  </div>
);

export default Guest;
