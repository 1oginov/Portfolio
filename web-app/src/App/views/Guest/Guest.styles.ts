import { withStyles } from '@material-ui/core';

import { viewHeadline, viewRoot } from 'Styles/mixins';

export default withStyles((theme) => ({
  headline: viewHeadline(theme),
  root: {
    ...viewRoot(theme),
    justifyContent: 'center',
  },
}));
