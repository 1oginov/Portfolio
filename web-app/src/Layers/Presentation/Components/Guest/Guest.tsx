import * as React from 'react';

import { GoogleLoginButton } from 'Layers/Presentation/Components/GoogleLoginButton';
import { Logo } from 'Layers/Presentation/Components/Logo';

interface Props {
  classes: { [key: string]: string };
}

export const Guest: React.FunctionComponent<Props> = ({ classes }: Props) => (
  <div className={classes.root}>

    <Logo className={classes.headline} />

    <GoogleLoginButton />

  </div>
);
