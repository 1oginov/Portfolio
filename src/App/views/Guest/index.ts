/* @flow */

import Guest from './Guest';

export default Guest;
