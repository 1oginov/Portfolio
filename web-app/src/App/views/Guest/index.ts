import Guest from './Guest';
import style from './Guest.styles';

export default style(Guest);
