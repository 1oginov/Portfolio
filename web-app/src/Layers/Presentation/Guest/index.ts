import { Guest as GuestComponent } from './Guest';
import { style } from './Guest.style';

export const Guest = style(GuestComponent);
