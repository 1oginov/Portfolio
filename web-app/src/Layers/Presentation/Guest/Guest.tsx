import * as React from 'react';

import { Logo } from 'Layers/Presentation/Logo';
import GoogleLoginButton from 'User/components/GoogleLoginButton';

interface Props {
  classes: { [key: string]: string };
}

export const Guest: React.FunctionComponent<Props> = ({ classes }: Props) => (
  <div className={classes.root}>

    <Logo className={classes.headline} />

    <GoogleLoginButton />

  </div>
);
