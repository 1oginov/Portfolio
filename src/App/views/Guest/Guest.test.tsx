import { shallow } from 'enzyme';
import classes from 'identity-obj-proxy';
import * as React from 'react';

import Guest from './Guest';

it('matches snapshot', () => {
  const wrapper = shallow(<Guest classes={classes} />);

  expect(wrapper).toMatchSnapshot();
});
