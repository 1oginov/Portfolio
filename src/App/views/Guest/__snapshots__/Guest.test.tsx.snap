// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="root"
>
  <div
    className="row"
  >
    <Logo />
  </div>
  <div
    className="row"
  >
    <GoogleLoginButton />
  </div>
</div>
`;
